### PR TITLE
Implemented to_ordinal for RU

### PR DESCRIPTION
**Pull request by smirnovvad**
### Changes proposed in this pull request:

* Implemented to_ordinal for RU
* Added test cases for this
### Status

- [X] READY
- [ ] HOLD
- [ ] WIP (Work-In-Progress)

### How to verify this change

Just run tests

### CODE REVIEW

File: num2words/lang_RU.py [hunk 1]
```diff
@@ -134,7 +155,42 @@ def pluralize(self, n, forms):
...
+                if lastword[-2] == "ч":
+                    lastword = lastword[:-1] + "ный"
+                if lastword[-1] == "ч":
+                    lastword = lastword + "ный"
+            elif lastword[-1] == "н" or lastword[-2] == "н":
+                lastword = lastword[:lastword.rfind('н') + 1] + "ный"
+            elif lastword[-1] == "д" or lastword[-2] == "д":
+                lastword = lastword[:lastword.rfind('д') + 1] + "ный"
+        outwords[-1] = self.title(lastword)
+        return " ".join(outwords).strip()
```
Comment (erozqba) [inline num2words/lang_RU.py]: @smirnovvad  There are some `print()` statements that it seems you used to debug. Are they needed? If not Could you remove them?

Comment (smirnovvad) [reply num2words/lang_RU.py]: Forgot about it, sorry :)

**Comment by erozqba** [comment]
@smirnovvad with only a couple more of tests we could have 100% percent coverage for RU. Could you include them to cover the missing lines https://coveralls.io/builds/19506908/source?filename=num2words%2Flang_RU.py#L187 ?

File: num2words/lang_RU.py (hunk 1 above)
Comment (erozqba) [reply num2words/lang_RU.py]: Thanks!

**Comment by smirnovvad** [comment]
In some rare cases russian lang can be very hard to predict 😟

**Comment by erozqba** [comment]
@smirnovvad it's too much to ask for a test to cover this line https://coveralls.io/builds/19507226/source?filename=num2words%2Flang_RU.py#L199 I know it's not related to your changes, so I will understand if you say you will not add another test.

**Comment by erozqba** [comment]
Thanks a lot, @smirnovvad :smile:

**Pull request merged**